src/openms/include/OpenMS/KERNEL/MRMTransitionGroup.h

```diff
@@ -335,6 +335,12 @@ namespace OpenMS
         }
       }
 
+      for (typename std::vector<SpectrumType>::iterator pr_it = precursor_chromatograms_.begin(); pr_it != precursor_chromatograms_.end(); ++pr_it)
```

Comment:
This is a good suggestion and might be very useful for future extensions. I'm wondering how we should best incorporate this. Would we need specific precursor IDs apart from GroupID_Precursor_i0? At the moment these are always derived from the group name, right? Or is there a different function for MS1-only mode?